### PR TITLE
fix(py): fix redis connection

### PR DESCRIPTION
**Pull request by eeliu**
## Describe your changes

- use format_host

## Issue ticket number and link

#612 

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [x] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

File: plugins/PY/pinpointPy/libs/_pyRedis/PyRedisPlugins.py [hunk 1]
```diff
@@ -26,6 +26,10 @@
 from pinpointPy import Defines
 
 
+def format_host(host, port, db) -> str:
```
Comment (eeliu) [inline plugins/PY/pinpointPy/libs/_pyRedis/PyRedisPlugins.py:29]: as the destination format must be the same in a plugin.

File: plugins/PY/pinpointPy/libs/_pyRedis/PyRedisPlugins.py [hunk 2]
```diff
@@ -62,8 +68,9 @@ def onBefore(self, parentId, *args, **kwargs):
             pipeLine = args[0]
             # fixed: Redis Collections Not release #612
             # @quicksandznzn
+            connection_kwargs = pipeLine.connection_pool.connection_kwargs
             pinpoint.add_trace_header(
```
Comment (quicksandznzn) [inline plugins/PY/pinpointPy/libs/_pyRedis/PyRedisPlugins.py:72]: must default value 
like this
` pinpoint.add_trace_header(
                Defines.PP_DESTINATION, format_host(connection_kwargs.get('host','localhost'),
                                                    connection_kwargs.get('port',6379),
                                                    connection_kwargs.get('db',0))
                , trace_id)`

Comment (quicksandznzn) [reply plugins/PY/pinpointPy/libs/_pyRedis/PyRedisPlugins.py:72]: @eeliu

Comment (quicksandznzn) [reply plugins/PY/pinpointPy/libs/_pyRedis/PyRedisPlugins.py:72]: I have a questions,  this dst value look at only show in apm web ,the java code call redis show this , python code why need host port db
image

Comment (quicksandznzn) [reply plugins/PY/pinpointPy/libs/_pyRedis/PyRedisPlugins.py:72]: If you don't mind. please add my wechat account **Quicksand_ZN**

Comment (eeliu) [reply plugins/PY/pinpointPy/libs/_pyRedis/PyRedisPlugins.py:72]: > I have a questions, this dst value look at only show in apm web ,the java code call redis show this , python code why need host port db image

Good question,

it's ok to keep the same format, but sometime user needs more, like port, db( mostly on redis cluster)

Comment (eeliu) [reply plugins/PY/pinpointPy/libs/_pyRedis/PyRedisPlugins.py:72]: > connection_kwargs

I will add them.

**Comment by quicksandznzn** [comment]
LGTM

**Pull request merged**